.pre-commit-config.yaml

```diff
@@ -15,6 +15,13 @@ repos:
         args: [--ignore-missing-imports]
         files: ^pymc_marketing/
         additional_dependencies: [numpy>=1.20, pandas-stubs]
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.7.1
+    hooks:
+    - id: nbqa-ruff
+      args: ["--fix", "--show-source"]
+      files: ^docs/source/notebooks/
+      exclude: ^docs/source/notebooks/clv/dev/
```

Comment:
These aren't on the documentation site anyways, right?

Reply:
No, the are not